pygsti/extras/interpygate/__init__.py

```diff
@@ -10,3 +10,10 @@
 
 from .core import PhysicalProcess, InterpolatedDenseOp, InterpolatedOpFactory
 from .process_tomography import vec, unvec, run_process_tomography
+
+# Note from Riley on May 22, 2024:
+#
+#   I wanted to remove the implementations of vec and unvec and just in-line equivalent 
+#   code in the few places they were used. However, the fact that they're included in this
+#   __init__.py file suggests that they might be used outside of pyGSTi itself.
+#
```

Comment:
We can check with @kevincyoung and @bruzic3, but I think I'm OK with inlining these in our source, deprecating the functions in pyGSTi, and then assisting/enforcing the relevant changes in the downstream code so that they can be removed if they are inefficient/not best practice.